### DISCUSSION TITLE
A few questions about the transition to version 2

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#1684] [Q&A]
1. How to become an admin again? On my server, I was the only user (admin), but by negligence I transferred myself to the status of a regular user, how to return myself to the admin again?
2. How to remove unused types of modulation in the interface? Not being an admin?

**Comment by magicbug** [comment]
1. easiest is edit the user table in the database change the roll to 99
2. only available currently to admin users and its a global setting

**Reply by MikleBas** [reply]
Yes, everything worked out! :)
Thank you very much for your help and your wonderful program!
PS the second version looks very good, works fast!
PPS where to look at the page that preceded the transition from version 1 to version 2? There were recommendations for the cron and now I can not find it ..

**Reply by magicbug** [reply]
You can find the list in the wiki https://github.com/magicbug/Cloudlog/wiki/Recommended-Cron-Jobs

**Reply by MikleBas** [reply]
And how to find out whether the Russian language is supported or not? If so, how to translate the appearance into the desired language?
Thanks

**Reply by magicbug** [reply]
Currently, we don't have Russian purely as no ones has done it! but PRs accepted against the dev branch :)

See https://github.com/magicbug/Cloudlog/tree/master/application/language

**Reply by R1BLH** [reply]
> Currently, we don't have Russian purely as no ones has done it! but PRs accepted against the dev branch :)

I done some translations, but i have no expirience with Github to make PR. :)
Also i done static buttons to open callsign page on qrz.ru.
